src/Components/List/List.js

```diff
@@ -1,10 +1,59 @@
 import React from "react";
+import Card from "./Card";
+import CardDiscount from "./CardDiscount";
 import "./List.scss";
 
 class List extends React.Component {
-  render() {
-    return <div className="List"></div>;
-  }
+    constructor() {
+        super();
+
+        this.state = {
+            productList: [],
+        };
+    }
+
+    componentDidMount() {
+        fetch("http://localhost:3000/Data/mock.json", {
+            method: "GET",
+        })
```

Comment:
```suggestion
        fetch("/Data/mock.json")
```
- 주소를 이렇게만 작성해도 루트 경로를 찾습니다! 오히려 이렇게 작성하는 것이 포트번호나 url이 바뀌었을 때 변수가 적습니다.
- method에서 get이 default값이기 때문에 생략해도 무방할거 같습니다.
- 데이터를 가져오는 메서드를 별도로 분리한 다음 그 메서드를 CDM에서 실행하는 방식이 가독성 차원에서 좋을 것 같습니다.